### DISCUSSION TITLE
How do I read values only from protobuf written using python protobuf as well

### DISCUSSION THREAD

**Discussion by RebbahD** [protobuf-c/protobuf-c#606] [Q&A]
Hello, 

I have a protobuf code in Python that writes my data to a .bin file, and I need to read that data from that .bin file using C, using the protobuf-c I did the generation of the libraries for C and protobuf --python to get the library for python for the same proto file. 

The writing is good however the reading I am facing so much issues. Please help me with it, As I cannot find a solution to this elsewhere.

**Comment by lipnitsk** [comment]
Please study the protobuf-c API and browse sample code. The library can unpack a serialized bytestream regardless of the originating platform, that is the whole purpose of using protobufs. This issue does not report a bug in protobuf-c, so closing it.
